src/renderer/components/atoms/checkbox/checkbox.tsx

```diff
@@ -13,6 +13,10 @@ const Checkbox = ({
   const [value, setValue] = useState<boolean>(checked || false);
   const inputId = id || `checkbox-${getRandomHex(16)}`;
 
+  useEffect(() => {
+    setValue(checked || false);
+  }, [checked]);
+
```

Comment:
I found a bug on this component and `SelectionList` too. I'm trying to implement this pattern where we have some props that, when used, allow us to control the component from an outside state. The issue is that the internal state is not getting updated automatically, so this `useEffect` is making sure of that

Reply:
nice!